miniapp/miniapp_reduction_to_band.cpp

```diff
@@ -38,17 +39,26 @@ struct Options
     : dlaf::miniapp::MiniappOptions<dlaf::miniapp::SupportReal::Yes, dlaf::miniapp::SupportComplex::Yes> {
   SizeType m;
   SizeType mb;
+  SizeType b;
 
   Options(const pika::program_options::variables_map& vm)
-      : MiniappOptions(vm), m(vm["matrix-size"].as<SizeType>()), mb(vm["block-size"].as<SizeType>()) {
+      : MiniappOptions(vm), m(vm["matrix-size"].as<SizeType>()), mb(vm["block-size"].as<SizeType>()),
+        b(vm["band-size"].as<SizeType>()) {
     DLAF_ASSERT(m > 0, m);
     DLAF_ASSERT(mb > 0, mb);
 
-    DLAF_ASSERT(do_check == dlaf::miniapp::CheckIterFreq::None,
-                "Error! At the moment result checking is not implemented. Please rerun with --check-result=none.");
+    if (b < 0)
+      b = mb;
+    DLAF_ASSERT(b > 0 && b <= mb, b, mb);
```

Comment:
Isn't the constrain `mb % b == 0` missing?
```suggestion
    DLAF_ASSERT(b > 0 && mb % b == 0, b, mb);
```

Reply:
Yeah, I was not sure about adding it or not.

My concern was that it is a limitation of the algorithm and I didn't want to relay it here, but it is also true that in miniapps we tend to constrain runs to what is possible to run currently.

I'm going to add it. 😄